utils/telegram.py

```diff
@@ -14,6 +14,22 @@ class Handlers:
     def start(update, context):
         context.bot.send_message(chat_id=update.message.chat_id, text=f'Olá, sou a Ada. O ID dessa conversa é {update.message.chat_id}. Se você é o administrador da Ada, sabe o que fazer.')
 
+    @staticmethod
+    def test(update, context):
+        button = InlineKeyboardButton('Deploy', callback_data='Test 123')
+        keyboard = InlineKeyboardMarkup.from_button(button)
+        bot = context.bot
+        bot.send_message(chat_id=update.message.chat_id, text='Isso é um teste de botão.', reply_markup=keyboard)
+
+
+def callback_handler(update, context):
+    bot = context.bot
+    query = update.callback_query
+    print('original message:', query.message.text)
+
+    bot.answer_callback_query(query.id, text='Deployed!')
+    bot.edit_message_text(chat_id=query.message.chat_id, message_id=query.message.message_id, text='Deployed!')
+
```

Comment:
This shouldn't be a part of the code yet. I won't take it off though, since it doesn't harm anyone.